src/components/learning-guide-chart.coffee

```diff
@@ -291,7 +301,7 @@ module.exports = class LearningGuideChart
         else if lv <= .49 and lv >= .35
           'yellow'
         else if lv < .35
-          'orange'
+          'grey'
```

Comment:
Should these be the semantic name `high/medium/low` or `ace/cruising/...` rather than the colors?